react-components/src/components/Architecture/SettingsButton.tsx

```diff
@@ -40,6 +40,14 @@ export const SettingsButton = ({
   const flexDirection = getFlexDirection(placement);
   const isTooltipDisabled = isOpen || label === undefined;
 
+  // This ensures the rule for where the dropdown panel should open:
+  // Horizontal toolbar at top:    Below the button
+  // Horizontal toolbar at bottom: Above the button
+  // Vertical toolbar at left:     Right of the button
+  // Vertical toolbar at right:    Left of the button
+  const dropdownPlacement =
+    placement === 'top' || placement === 'bottom' ? 'bottom-end' : 'right-start';
```

Comment:
Not sure I understand if the code does what the comment says :thinking: Won't this place the tooltip at `bottom-end` both when placement is manually set to `top` or `bottom`?